enterprise/dev/ci/internal/ci/operations.go

```diff
@@ -593,7 +590,8 @@ func clusterQA(candidateTag string) operations.Operation {
 			bk.Env("TEST_USER_PASSWORD", "supersecurepassword"),
 			bk.Env("INCLUDE_ADMIN_ONBOARDING", "false"),
 			bk.Cmd("./dev/ci/integration/cluster/run.sh"),
-			bk.ArtifactPaths("./*.png", "./*.mp4", "./*.log"))
+			bk.ArtifactPaths("./*.png", "./*.mp4", "./*.log"),
+			bk.SoftFail(1, 2))
```

Comment:
based on the above comment:

```suggestion
			// Flakey test we are running to collect more data:
			// https://github.com/sourcegraph/sourcegraph/issues/31342
			bk.SoftFail(123))
```